### ISSUE TITLE
How to solve the humanized time issue?

### ISSUE THREAD

**Issue by idc77** [bigskysoftware/htmx#1851]
Let's take a simple data type which has a date/time field in it.

Let's say I have a hx-post form which returns a html fragment with a newly created entity.
Let's also say, the time field of that entity has a "timeago" or humanized value.
The humanized value will always return "now", and the previously created entities will also have the value "now" until refresh of that page, if the entities created are stored.

Is there some kind of post swap script triggering mechanism in htmx?

There I created a repo demonstrating the problem.
https://code.icod.de/dalu/htmxginpongo

Before I forget, there's also another issue in this repo.
The table tags and tr th tags should only be rendered if there actually are messages.
The only solution I can think of is  re-render the whole document or at least the messages part and return all messages.
Now imagine that will a facebook type message list... 100s or 1000s of messages that need to be re-rendered.

I honestly fail to see how htmx would be able to tackle this kind of problem.
And all with real-time updating timestamps.

Of course you can say use the right tool for the right job.
But well... IMHO it complicates everything or you have to make compromises like using absolute time vs humanized time etc. 

Long story short, does a post response/apply script execution mechanism exist in htmx? "post response transform"

**Comment by alexpetros** [comment]
> Long story short, does a post response/apply script execution mechanism exist in htmx? "post response transform"

Yup, check out `hx-on:`. You can run a script at any point in the event lifecycle to modify the htmx request or run any arbitrary code.
